src/HtmlSanitizer/HtmlSanitizer.cs

```diff
@@ -578,6 +578,21 @@ public IHtmlDocument SanitizeDom(string html, string baseUrl = "")
             return dom;
         }
 
+        /// <summary>
+        /// Sanitizes the specified parsed HTML body fragment.
+        /// The Document Must have been pared with CSS support and the following options enabled
+        /// "IsIncludingUnknownDeclarations", "IsIncludingUnknownRules" and "IsToleratingInvalidSelectors"
```

Comment:
What happens if you pass a document that was not parsed using these options?

Reply:
If you are not using `WithCss` then IElement.GetStyle() will always return null meaning all style attributes are stripped. I think it will likewise effect parsing style sheets although i have not tested that. 

Looks like `document.Context.GetCssStyling();` well be null if css has not been configured so that might be worth throwing if its null? 

As for the individual options i believe angle sharp will strip the unrecognised part when parsing them, in which case this could be removed? As it should not effect the Sanitizer's operation.

Reply:
It's the caller's problem if all style will be stripped unless specific parsing options were used. Does something throw in HtmlSanitizer if these options are left out?

Reply:
For the style attribute it will fail silently, im not sure about style sheets.

Reply:
The unit tests run fine w/o these options. It seems they are only there for legacy reasons. If you leave out CSS support a number of tests fail of course but none throw (except the threads test but that's expected as it's just a meta test). So I think we can remove the comment.

Reply:
Ok i have removed the last line of the comment